packages/intrepid2/src/Orientation/Intrepid2_OrientationDef.hpp

```diff
@@ -97,8 +97,8 @@ namespace Intrepid2 {
     for(ordinal_type i=0;i<numVerts-1;++i)
       for(ordinal_type j=i+1;j<numVerts;++j)
         INTREPID2_TEST_FOR_ABORT( ( subCellVerts[i] == subCellVerts[j] ), 
-                        ">>> ERROR (Intrepid::Orientation::getOrientation): " \
-                        "Invalid subCellVerts, same vertex ids are repeated");
+                                 ">>> ERROR (Intrepid::Orientation::getOrientation): " \
+                                 "Invalid subCellVerts, same vertex ids are repeated");
```

Comment:
I saw you correct this to "some vertex ids" in other places

Reply:
Ah, thanks for catching this.  I think that correction got clobbered when I merged latest develop into this branch.